public/app/plugins/datasource/loki/backendResultTransformer.ts

```diff
@@ -1,8 +1,9 @@
 import { DataQueryResponse, DataFrame, isDataFrame, FieldType, QueryResultMeta } from '@grafana/data';
```

Comment:
the changes in this file are:
-  calling `getDerivedFields` at the correct place
- `getDerivedFields` needs the derived-field-configs, so we pass it along function calls